Mol_Analysis.py

```diff
@@ -645,6 +669,38 @@ def plotall(self, groupfolderlist, local=True, ncols=3, nrows=2, **kwargs):
 
 ### Functions
 
+def mindist_slice(args):
```

Comment:
see previous comment re `pool.starmap` - here you can just have arguments instead of tuple that you need to process

Reply:
it would be cleaner if you moved the parsing and removing of the output file to this function - so you just get the data you need, and it can be done in parallel